### PR TITLE
git-graft: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For #5137

### CODE REVIEW

File: pages/common/git-graft.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# git graft
+
+> Merge commits from a specific branch into another and delete the source branch.
```
Comment (bl-ue) [inline pages/common/git-graft.md:3]:
```suggestion
> Merge commits from a branch into another branch and delete the source branch.
```

File: pages/common/git-graft.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-graft>.
+
+- Merge all commits not present on destination branch from source branch, and delete source branch:
```
Comment (bl-ue) [inline pages/common/git-graft.md:7]:
```suggestion
- Merge all commits not present on the target branch from the source branch, and delete the source branch:
```

File: pages/common/git-graft.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+`git graft {{source_branch}} {{destination_branch}}`
```
Comment (bl-ue) [inline pages/common/git-graft.md:9]:
```suggestion
`git graft {{source_branch}} {{target_branch}}`
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
"Target" is more common than "destination."

File: pages/common/git-graft.md (hunk 1 above)
Comment (CleanMachine1) [reply pages/common/git-graft.md:3]: I like the specific, how about that

Comment (bl-ue) [reply pages/common/git-graft.md:3]: Updated.

File: pages/common/git-graft.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# git graft
+
+> Merge commits from a specific branch into another branch and delete the source branch.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-graft>.
+
+- Merge all commits not present on the target branch from the source branch, and delete the source branch:
```
Comment (bl-ue) [inline pages/common/git-graft.md:7]: This makes it longer, but re-reading it, without this extra bit the sentence sounds incomplete.
```suggestion
- Merge all commits not present on the target branch from the source branch to target branch, and delete the source branch:
```

Comment (marchersimon) [reply pages/common/git-graft.md]: Broken down it would be `merge sth from sth`. To me it doesn't sound that bad.

**Review by bl-ue — APPROVED** [review]

File: pages/common/git-graft.md (hunk 4 above)
Comment (marchersimon) [reply pages/common/git-graft.md]: Doesn't matter, with the three extra words it's definetely clearer.

**Review by marchersimon — APPROVED** [review]

File: pages/common/git-graft.md (hunk 4 above)
Comment (bl-ue) [reply pages/common/git-graft.md]: What does `sth` mean there, @marchersimon? I'm confused.

Comment (marchersimon) [reply pages/common/git-graft.md]: Isn't `sth` a common abbreviation of `something` and a placeholder if you want to describe how a verb can be used? (At least that's how most dictionaries do it.)

Comment (CleanMachine1) [reply pages/common/git-graft.md]: DId someone unresolve this or did I not commit it?

Comment (marchersimon) [reply pages/common/git-graft.md]: I think @bl-ue unresolved. You committed it in e0298f2.

Comment (bl-ue) [reply pages/common/git-graft.md]: Ah @marchersimon I see!  I've not seen `sth` yet, I've only seen `smth` 🤔

**Review by navarroaxel — APPROVED** [review]

**Comment by CleanMachine1** [comment]
24hours passed, main 3 reviewers reviewed. Merged

**Pull request merged**